### DISCUSSION TITLE
413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by Ganesh1991** [hasura/graphql-engine#8849] [Question] [answered]
### Version Information

Server Version: v2.0.7

### What is the current behaviour?

I have created a function that takes a JSON array of objects as input. While calling the function from the Hasura console getting the below error for the input of size 3.46MB 

`GraphQL Error (Code: 413): {"response":{"error":"<html>\r\n<head><title>413 Request Entity Too Large</title></head>\r\n<body>\r\n<center><h1>413 Request Entity Too Large</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n","status":413,"headers":{}}

Not able to find any details on where can I set the limit in the console against the input.

Any suggestions???

**Comment by tirumaraiselvan** [comment]
Where is your Hasura hosted? The error seems to be coming from nginx...

**Comment by Ganesh1991** [comment]
Hasura is hosted on kubernetes.

**Comment by jasonmccallister** [comment, marked as answer]
@Ganesh1991 you will need to update the maximum size that nginx is allowed to handle. This unfortunately is not something Hasura controls. Also, 3mb is a really large JSON payload that might be worth reducing into more than one request.

https://medium.com/@svsh227/error-413-request-entity-too-large-in-nginx-with-client-max-body-size-changes-in-nginx-6aacd525fe11
